lib/sycamore/sycamore/context.py

```diff
@@ -109,7 +117,11 @@ def deregister_rule(self, rule: Rule) -> None:
 _global_context: Optional[Context] = None
 
 
-def init(exec_mode=ExecMode.RAY, ray_args: Optional[dict[str, Any]] = None) -> Context:
+def init(exec_mode=ExecMode.RAY, ray_args: Optional[dict[str, Any]] = None, config: Optional[Config] = None) -> Context:
+    """
+    Initialized a new Context. If there is already an initialized Context, we reuse the Ray session but allow you to
+    override any Config variables.
+    """
```

Comment:
ooh actual documentation for sycamore.init()! (nit: Initialized -> Initialize; and maybe add an Args: block)

Reply:
Maybe add this module to somewhere in the api docs toctree, idk